platform/javascript/export/export_plugin.cpp

```diff
@@ -142,12 +143,18 @@ void EditorExportPlatformJavaScript::_fix_html(Vector<uint8_t> &p_html, const Re
 		head_include += "<script type='application/javascript'>window.addEventListener('load', () => {if ('serviceWorker' in navigator) {navigator.serviceWorker.register('" +
 				p_name + ".service.worker.js');}});</script>\n";
 	}
-
+	else {
```

Comment:
Style error, this should be `} else {`. See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html